test/__init__.py

```diff
@@ -146,6 +151,18 @@ def notBrotli() -> Callable[[_TestFuncT], _TestFuncT]:
     )
 
 
+def onlyZstd() -> Callable[[_TestFuncT], _TestFuncT]:
+    return pytest.mark.skipif(
+        zstd is None, reason="only run if zstandard library is present"
+    )
+
+
+def notZstd() -> Callable[[_TestFuncT], _TestFuncT]:
+    return pytest.mark.skipif(
+        zstd is not None, reason="only run if a zstandard library is absent"
```

Comment:
Unlike brotlipy, we only support one Zstandard library:

```suggestion
        zstd is None, reason="only run if python-zstandard is installed"
```